forest/daemon/src/bundle/mod.rs

```diff
@@ -20,12 +20,12 @@ where
 {
     // collect bundles to load into the database.
     let mut bundles = Vec::new();
-    if epoch < config.chain.epoch(Height::Hygge) {
-        bundles.push(get_actors_bundle(config, Height::Hygge).await?);
-    }
-    if epoch < config.chain.epoch(Height::Lightning) {
-        bundles.push(get_actors_bundle(config, Height::Lightning).await?);
+    for info in &config.chain.height_infos {
+        if info.bundle.is_some() && epoch < config.chain.epoch(info.height) {
+            bundles.push(get_actors_bundle(config, info.height).await?);
+        }
     }
+
     // Nothing to do regarding Thunder since it's more like a "ghost" upgrade.
```

Comment:
This comment could be moved to the place where the Thunder `HeightInfo` is defined.

Reply:
Done